db/migrate.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Please make sure that the web application does not have any access to
```

Comment:
Remove this comment. It is incomprehensible.

Reply:
Fixed in next push